plugin/agent/workloadattestor-unix/unix_test.go

```diff
@@ -20,6 +21,10 @@ func TestUnix_AttestValidPID(t *testing.T) {
 }
 
 func TestUnix_AttestInvalidPID(t *testing.T) {
+	switch runtime.GOOS {
+	case "darwin":
+		t.Skip("skipping test on %v", runtime.GOOS)
```

Comment:
Hmm I thought the library we were using for this was relatively portable - I assume you hit an error here or something?

Reply:
oh i did not mean to add this to the commit... but since its here, all PIDs including -1 are valid on darwin, so this test would never fail(succeed)

Reply:
Oh, interesting... A comment to that effect would be useful, but not a blocker. I can re-run the tests here once 236 is merged